apps/innovations/v1-innovation-reassessment-request-create/validation.schemas.ts

```diff
@@ -11,19 +10,15 @@ export const ParamsSchema = Joi.object<ParamsType>({
 }).required();
 
 export type BodyType = {
-  updatedInnovationRecord?: YesOrNoCatalogueType;
+  reassessmentReason: string[];
+  otherReassessmentReason?: string;
   description: string;
+  whatSupportDoYouNeed: string;
 };
 
 export const BodySchema = Joi.object<BodyType>({
-  description: Joi.string().max(TEXTAREA_LENGTH_LIMIT.xl).required()
-}).when('$userRole', [
-  {
-    is: ServiceRoleEnum.INNOVATOR,
-    then: Joi.object({
-      updatedInnovationRecord: Joi.string()
-        .valid(...Object.values(YesOrNoCatalogueType))
-        .required()
-    }).required()
-  }
-]);
+  reassessmentReason: Joi.array().items(Joi.string()).min(1).required(),
```

Comment:
If you add the type of valid values, in here you should validate that only valid items are sent.

Reply:
Will do.